src/main/kotlin/com/quipalup/katydid/logentry/application/UpdateLogEntryByIdCommandHandler.kt

```diff
@@ -0,0 +1,51 @@
+package com.quipalup.katydid.logentry.application
+
+import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.left
+import arrow.core.right
+import com.quipalup.katydid.common.id.Id
+import com.quipalup.katydid.logentry.domain.FindLogEntryError
+import com.quipalup.katydid.logentry.domain.LogEntry
+import com.quipalup.katydid.logentry.domain.LogEntryRepository
+import com.quipalup.katydid.logentry.domain.SaveLogEntryError
+import com.quipalup.katydid.logentry.primaryadapter.httprest.LogEntryUpdateAttributes
+import java.util.UUID
+import javax.inject.Named
+
+@Named
+class UpdateLogEntryByIdCommandHandler(private val logEntryRepository: LogEntryRepository) {
+    fun execute(command: UpdateLogEntryByIdCommand): Either<SaveLogEntryError, Id> =
```

Comment:
```suggestion
    fun execute(command: UpdateLogEntryByIdCommand): Either<UpdateLogEntryByIdError, Id> =
```
Each use case shoud have its own sealed class of errors.